src/install/config/kubernetes.yaml

```diff
@@ -53,6 +53,13 @@ steps:
             - optionType: dropdown1
               options: 
                 - value: 'manifest'
+  - filePath: 'src/install/kubernetes/guided/configuration-guided.mdx'
```

Comment:
@nbaenam can you tell me how to test for this condition in the docs?

Also, I found this sentence that seemed awkward: "Create a file named yaml file, for example `newrelic-manifest.yaml`. Include the configuration you need."

How about this: "Create a YAML configuration file, for example `newrelic-manifest.yaml`. Include the configurations you need, and make sure it follows this structure:"

Reply:
Done.